file_paths.md

```diff
@@ -220,99 +215,132 @@ create a `puppet` user or group.
             puppet.conf *                     # :config
             routes.yaml                       # :route_file
             ssl                               # :ssldir
-    -------------------------------------------------------------------------------------------------------
-        var *
-            log                               # :logdir                      same
-                puppet.log                    # not enabled by default
-            run                               # :rundir                      same
-                agent.pid                     # :pidfile
 
     C:\ProgramData\PuppetLabs\pxp-agent *
         etc *
             pxp-agent.conf                    # pxp-agent configuration file
             modules *                         # stores configuration files for pxp-agent modules
-                pxp-module-puppet.conf        # configuration file of the pxp module puppet (optionally, to override puppet.bat location)
+                pxp-module-puppet.conf *      # configuration file of the pxp module puppet (optionally, to override puppet.bat location)
         var *
-            log *
-                pxp-agent.log                 # enabled by default
             spool *                           # directory containing results of pxp-agent modules
-            run *
-
-    C:\Program Files\Puppet Labs\Puppet
-        VERSION                               # puppet-agent package version
-
-    C:\Program Files\Puppet Labs\Puppet\bin *
-        environment.bat *                     # setup LOAD_PATH
-        facter.bat *                          # bat file wrappers
-        hiera.bat *
-        mco.bat *
-        puppet.bat *
-        puppet_shell.bat *                    # targets for shortcuts
-        run_facter_interactive.bat *
-        run_puppet_interactive.bat *
-
-    C:\Program Files\Puppet Labs\Puppet\facter *
-        bin *                                 # executables and dlls
-            facter.exe *
-            libfacter.so *
-            lib*.dll *                        # gcc, mingw, boost libraries
-        inc *                                 # facter headers
-            facter *
-        lib *
-            facter.rb *                       # ruby bindings
-
-    C:\Program Files\Puppet Labs\Puppet\hiera *
-        bin *
-            hiera *                           # ruby bin wrapper
-        lib *
-            hiera.rb *
 
-    C:\Program Files\Puppet Labs\Puppet\mcollective *
-        bin *
-            mcollectived *                    # ruby bin wrapper
-            mco *
-        lib *
-            mcollective.rb *
+    C:\ProgramData\PuppetLabs\var
```

Comment:
@ferventcoder I think you gave this move a thumbs up, but since it's in ProgramData, is it something we can do? Or do these pidfiles/logfiles need to be where they originally were?